packages/react-components/src/molecules/ListControls.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { NavHashLink } from 'react-router-hash-link';
+import css from '@emotion/css';
+
+import { charcoal, fern, lead } from '../colors';
+import { perRem, lineHeight } from '../pixels';
+import { cardViewIcon, listViewIcon } from '../icons';
+
+const activeClassName = 'active-link';
+
+const containerStyles = css({
+  display: 'grid',
+  gridAutoFlow: 'column',
+  columnGap: '24px',
+  width: 'fit-content',
+  margin: `${12 / perRem}em 0`,
+});
+
+const styles = css({
+  color: charcoal.rgb,
+  textDecoration: 'none',
+  outline: 'none',
+});
+
+const textStyles = css({
+  margin: 0,
+  display: 'flex',
+  fontSize: `${18 / perRem}em`,
+  svg: {
+    stroke: lead.rgb,
+  },
+});
+const iconStyles = css({
+  display: 'inline-block',
+  width: `${lineHeight / perRem}em`,
+  height: `${lineHeight / perRem}em`,
+  paddingRight: `${14 / perRem}em`,
+});
+
+const activeStyles = css({
+  fontWeight: 'bold',
+  svg: {
+    stroke: fern.rgb,
+  },
+});
+
+interface ListControlsProps {
+  readonly listView: boolean;
+  readonly listViewHref: string;
+  readonly cardViewHref: string;
+}
+const ListControls: React.FC<ListControlsProps> = ({
+  listView,
+  listViewHref,
+  cardViewHref,
+}) => (
+  <div css={containerStyles}>
+    <NavHashLink
+      to={cardViewHref}
+      activeClassName={activeClassName}
+      css={[styles, { [`&.${activeClassName}`]: activeStyles }]}
+      smooth
+      isActive={() => !listView}
```

Comment:
why is this necessary? The nav link should do that by default based on `to`

Reply:
that would also reduce props to two

Reply:
The URL will change with pagination and searchQuery params. I don't know how it would match without this

Reply:
right, and search query is preserved, but current page isn't I assume?

Reply:
The thing is, then we don't actually want a `NavLink`, but just a normal `Link`

Reply:
What I would suggest instead though: Still use `NavLink`, and still omit the `listView` prop, but in `isActive` look at that one specific `viewParam` and compare it to determine the active state

Reply:
That should mean little change from now because we're still using `NavLink`, but in a way that makes sense (you can tell because it actually uses the param to the isActive callback :D), and we still manage to omit one prop